### ISSUE TITLE
objective support mean_absolute_error?

### ISSUE THREAD

**Issue by casertillo** [EducationalTestingService/skll#270]
Hi,

Does objective for regressions support the metric.mean_absolute_error? http://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_error.html?

Thanks!
-Sergio.

**Comment by aoifecahill** [comment]
Yes, it should since it is part of ``sklearn.metrics.SCORERS``.
